riscv/processor.cc

```diff
@@ -1444,12 +1444,12 @@ reg_t processor_t::get_csr(int which, insn_t insn, bool write, bool peek)
   uint32_t ctr_en = -1;
   if (state.prv < PRV_M)
     ctr_en &= state.mcounteren;
+  if (state.v && state.prv <= PRV_S)
```

Comment:
Is it ever possible to have `state.v && state.prv > PRV_S`? i.e. I don't see the need for the check of `state.prv` here.

Reply:
Agreed.

Reply:
`state.v && state.prv > PRV_S` is not possible.

The additional `state.prv <= PRV_S` check is totally redundant. I will remove it.